setup.bzl

```diff
@@ -66,14 +67,14 @@ def kythe_rule_repositories():
     )
 
     maybe(
-        http_archive,
+        git_repository,
```

Comment:
Use `http_archive` to fetch a specific commit archive if necessary; `git_repository` is wastful of bandwidth and time as it unnecessarily clones the entire repository rather than a specific commit.

Reply:
I had some bizarre issues with the released bit - happy to rethink it now that things generally work.

Reply:
Reverted.